### PR TITLE
Feat: Added Navbar

### PR DESCRIPTION
**Pull request by fahd23**
- Added Home Page

### CODE REVIEW

File: src/components/banner/Banner.jsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import "components/banner/banner.css";
+
+const Banner = () => {
+  return (
+    <>
```
Comment (JantuDeb) [inline src/components/banner/Banner.jsx:6]: You can remove this since, `div` is the only child it has

**Review by JantuDeb — APPROVED** [review]
Suggested a changes

**Review by ShreyashMahajan — COMMENTED** [review]
good to merge

**Review by ShreyashMahajan — APPROVED** [review]
code looks well structured and neat. Good to go

File: src/components/banner/Banner.jsx (hunk 1 above)
Comment (fahd23) [reply src/components/banner/Banner.jsx:6]: will remove it, thanks for the feedback.

**Pull request merged**